release-notes.txt

```diff
@@ -1,3 +1,366 @@
+RIOT-2019.04 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2019.04 release includes a number of new features including improved
+support for ESP boards, porting of riotboot to a number of new platforms,
+802.15.4 support on the nRF52, and the ability for firmware to flash images
+into a separate boot slot. Support for several new boards and new sensors was
+added, and this release contains a number of bug fixes and test improvements.
+
+About xx pull requests with about yy commits have been merged since the last
+release and about zz issues have been solved. aa people contributed with code
+in bb days. Approximately cc files have been touched with dd insertions
+and ee deletions.
+
+
+Notations used below:
+=====================
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+New features and changes
+========================
+
+System libraries
+----------------
+    + riotboot_flashwrite: added module to reliably write a firmware image to
+      flash
+    + crypto: added support for chacha20poly1305 AEAD symmetric key cypher
+      algorithm
+    * threads: added usage of thread_state_t enum to define thread states,
+      instead of an integer
+    * random: added usage of hwrng as seed if available
+    + app_metadata: added module to print application metadata
+    + unaligned.h: added functions to safely read values from potentially
+      unaligned pointers
+
+Networking
+----------
+    * gnrc_ipv6: adapted ping6 to original-ping-like implementation
+    * gnrc_ipv6: allowed sending empty IPv6 packets
+    - gnrc_pktbuf: removed deprecated gnrc_pktbuf_get_iovec() function
+    + NDP: allow definition of common hop limit
```

Comment:
What is this referring to?

Reply:
If it refers to https://github.com/RIOT-OS/RIOT/pull/11011, this isn't really something that must be mentioned in the release notes. It's not a feature, it's just an introduction of a name instead of a magic number ;-).